OpenTabletDriver.Daemon/DriverDaemon.cs

```diff
@@ -141,6 +141,10 @@ public async Task<TabletState> DetectTablets()
 
         public Task SetSettings(Settings settings)
         {
+            foreach (var interpolator in Driver.Interpolators)
+                interpolator.Dispose();
+            Driver.Interpolators.Clear();
```

Comment:
Any reasoning for moving this out of the `SetInterpolatorSettings()` method? This is kind of hard to justify.

Reply:
Add this comment and I'll approve
```suggestion
            // Dispose all interpolators to begin changing settings
            foreach (var interpolator in Driver.Interpolators)
                interpolator.Dispose();
            Driver.Interpolators.Clear();
```